src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -129,9 +227,54 @@ private string GenerateJsonStr(EventWrittenEventArgs eventData)
                 json.Add("RelatedActivityId", eventData.RelatedActivityId);
             }
 
-            // Generate string-representation of JSON. Also remove newlines.
-            string jsonString = json.ToString(Newtonsoft.Json.Formatting.None);
-            jsonString = jsonString.Replace("\n", "\\n").Replace("\r", "\\r");
+            string jsonString = "";
```

Comment:
Since we might actually be writing a comma-separated string, I recommend not calling this `jsonString`. It makes reading the code confusing.